[2013-09-27 14:33:01.579000]
ERROR
	Peer will receive following PB traceback:
[2013-09-27 14:33:01.752000]
ERROR
	Unhandled Error
Traceback (most recent call last):
  File "D:\Program Files\pythonexe\lib\site-packages\twisted\spread\banana.py", line 153, in gotItem
    self.callExpressionReceived(item)
  File "D:\Program Files\pythonexe\lib\site-packages\twisted\spread\banana.py", line 116, in callExpressionReceived
    self.expressionReceived(obj)
  File "D:\Program Files\pythonexe\lib\site-packages\twisted\spread\pb.py", line 584, in expressionReceived
    method(*sexp[1:])
  File "D:\Program Files\pythonexe\lib\site-packages\twisted\spread\pb.py", line 896, in proto_message
    self._recvMessage(self.localObjectForID, requestID, objectID, message, answerRequired, netArgs, netKw)
--- <exception caught here> ---
  File "D:\Program Files\pythonexe\lib\site-packages\twisted\spread\pb.py", line 910, in _recvMessage
    netResult = object.remoteMessageReceived(self, message, netArgs, netKw)
  File "D:\Program Files\pythonexe\lib\site-packages\twisted\spread\flavors.py", line 114, in remoteMessageReceived
    state = method(*args, **kw)
  File "D:\Program Files\pythonexe\lib\site-packages\firefly-1.2.2dev-py2.6.egg\firefly\distributed\reference.py", line 26, in remote_callChild
    return self._service.callTarget(command,*arg,**kw)
  File "D:\Program Files\pythonexe\lib\site-packages\firefly-1.2.2dev-py2.6.egg\firefly\utils\services.py", line 84, in callTarget
    result = self.callTargetSingle(targetKey,*args,**kw)
  File "D:\Program Files\pythonexe\lib\site-packages\firefly-1.2.2dev-py2.6.egg\firefly\utils\services.py", line 104, in callTargetSingle
    defer_data = target(*args,**kw)
  File "E:\Py\sanguo - v1.70\app\scense\nodeapp\shop.py", line 95, in buyItemInMall_214
    data = shop.buyItemInMall(dynamicId, characterId, itemTemplateId,count,priceType,buyType,presentName)
  File "E:\Py\sanguo - v1.70\app\scense\applyInterface\shop.py", line 84, in buyItemInMall
    return buyService(player, itemTemplateId, count, mall_item,tocid)
  File "E:\Py\sanguo - v1.70\app\scense\applyInterface\shop.py", line 96, in buyService
    flg= buyServicepack(player,mall_item,tocid)
  File "E:\Py\sanguo - v1.70\app\scense\applyInterface\shop.py", line 130, in buyServicepack
    result=player.pack.putNewItemsInPackage(newitem,count)
  File "E:\Py\sanguo - v1.70\app\scense\component\pack\CharacterPackComponent.py", line 424, in putNewItemsInPackage
    if item.baseInfo.getitemPage()==4:
  File "E:\Py\sanguo - v1.70\app\scense\component\baseInfo\ItemBaseInfoComponent.py", line 41, in getitemPage
    return self.getItemTemplateInfo().get("itemPage",0)
  File "E:\Py\sanguo - v1.70\app\scense\component\baseInfo\ItemBaseInfoComponent.py", line 62, in getItemTemplateInfo
    self._baseName = dbaccess.all_ItemTemplate[self.itemTemplateId]['name']
exceptions.KeyError: 50000041L

